pkg/cmd/list.go

```diff
@@ -31,14 +34,98 @@ type whoCan struct {
 
 	namespaces         clientcore.NamespaceInterface
 	rbac               clientrbac.RbacV1Interface
-	namespaceValidator whocan.NamespaceValidator
-	resourceResolver   whocan.ResourceResolver
-	accessChecker      whocan.APIAccessChecker
+	namespaceValidator NamespaceValidator
+	resourceResolver   ResourceResolver
+	accessChecker      APIAccessChecker
 
 	r           roles
 	apiResource meta.APIResource
 }
 
+func NewCmdWhoCan() *cobra.Command {
+	var namespaceFlag string
+	var allNamespaces bool
+
+	configFlags := genericclioptions.NewConfigFlags(true)
+
+	cmd := &cobra.Command{
+		Use:   "kubectl-who-can VERB TYPE [NAME]",
+		Short: "who-can shows which users, groups and service accounts can perform a given action",
+		Long:  "who-can shows which users, groups and service accounts can perform a given verb on a given resource type",
+		Example: `  # List who can get pods in any namespace
+  kubectl who-can get pods
```

Comment:
I think this example is now listing pods in the current namespace? Might be nice to have an example for all namespaces as well, wdyt?

Reply:
Good catch @lizrice . I've updated the examples.